dapr/proto/runtime/v1/dapr_additional.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtime
+
+import (
+	"google.golang.org/protobuf/proto"
+
+	commonv1pb "github.com/dapr/go-sdk/dapr/proto/common/v1"
+)
+
+// CryptoRequests is an interface for EncryptRequest and DecryptRequest.
+type CryptoRequests interface {
+	proto.Message
+
+	// SetPayload sets the payload.
+	SetPayload(payload *commonv1pb.StreamPayload)
```

Comment:
Having these interfaces allow me to implement `performCryptoOperation` just once for both encrypt and decrypt. Otherwise the code has to be implemented twice.